### PR TITLE
status-data

### PR DESCRIPTION
**Pull request by burgerni10**
This PR start to remove statusData from connectors to be managed in Protocol / Api Handlers. 
I'll go further when PR #1797 will be done

### CODE REVIEW

File: src/north/MongoDB/MongoDB.class.js [hunk 1]
```diff
@@ -56,10 +56,11 @@ class MongoDB extends ApiHandler {
     this.logger.trace(`MongoDB handleValues() call with ${values.length} values`)
     try {
       await this.makeRequest(values)
-      this.statusData['Last handled values at'] = new Date().toISOString()
-      this.statusData['Number of values sent since OIBus has started'] += values.length
-      this.statusData['Last added point id (value)'] = `${values[values.length - 1].pointId} (${JSON.stringify(values[values.length - 1].data)})`
-      this.updateStatusDataStream()
+      this.updateStatusDataStream({
```
Comment (jfhenon) [inline src/north/MongoDB/MongoDB.class.js:59]: Most handleValues return the same information should the sse call be  in the parent class ?

**Review by jfhenon — COMMENTED** [review]
See comment

File: src/north/MongoDB/MongoDB.class.js (hunk 1 above)
Comment (burgerni10) [reply src/north/MongoDB/MongoDB.class.js:59]: I agree, but I don't think it should be done here, rather in the cache refactor in PR #1797

File: src/north/CsvToHttp/CsvToHttp.class.js [hunk 2]
```diff
@@ -123,7 +123,11 @@ class CsvToHttp extends ApiHandler {
           this.statusData['Last Parsed file'] = filePath
```
Comment (jfhenon) [inline src/north/CsvToHttp/CsvToHttp.class.js:123]: should be removed

**Review by jfhenon — CHANGES_REQUESTED** [review]

**Pull request merged**